File: JavaScript/Exercicios - FACCAT/Exercicio-19/exercicio19.js
// Exercícios, Lógica de Programação, Algoritmos e JavaScript
// Ler dois valores (considere que não serão lidos valores iguais) e escrever o maior deles.

// Campo do exercício 19
let valor1, valor2; //variável

alert("Qual número é maior") //saída de dados

valor1=parseInt(prompt("Digite um número: ")); //entrada e saída de dados
valor2=parseInt(prompt("Digite um número: ")); //entrada e saída de dados

if (valor1 == valor2){
    alert("Digite números DIFERENTES!");
}else if (valor1 > valor2) { //Processamento
    alert("O maior número é: "+valor1);  //saída de dados
}else{
    alert("O maior número é: "+valor2);  //saída de dados
}; 

File: Java/Exercicios - FACCAT/Exercicio19.java
package javaBasico;

import java.util.Scanner;

public class Exercicio19 {
	
	public static void main(String[] args) {
		
		Scanner dados = new Scanner(System.in);
		
		System.out.println("Qual número é maior");
		
		System.out.println("Digite um número: ");
		int valor1 = dados.nextInt();
		System.out.println("Digite um número: ");
		int valor2 = dados.nextInt();
		
		if (valor1 == valor2) {
			System.out.println("Digite números DIFERENTES!");
		}else if (valor1 > valor2) {
			System.out.println("O maior número é: "+valor1);
		}else {
			System.out.println("O maior número é: "+valor2);
		}
		
		dados.close();
	}
}

// Exercícios, Lógica de Programação, Algoritmos e JavaScript
/* Ler dois valores (considere que não serão lidos valores iguais) e escrever o maior deles. */
